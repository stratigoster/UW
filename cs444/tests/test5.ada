PACKAGE foo IS
  i : INTEGER := 3+4*5;
  b2 : BOOLEAN := i OR ELSE 5=5 AND THEN 3/3 = 3/3;
--                ^
-- OR ELSE expects bool, given int
END foo;
