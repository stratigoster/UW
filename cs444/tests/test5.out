***ERROR***:3:21:	OR-ELSE  works on boolean values

------
Compilation error
------
